Script started on 2020-05-28 11:10:49+0000
python /home/ubuntu/laspdev/utility/setup_lasp.py d1
root@csst-06:/home/ubuntu/laspdev/utility# python /home/ubuntu/laspdev/utility/setup_llasp.py d1
d1
ip:10.0.0.250
node:a
)2> 11:11:35.875 [warning] global: 'a@10.0.0.250' failed to connect to 'd@11.0.0.253'
11:11:42.666 [error] LASPVIN Recevied delta took: 74915 microseconds at TimeStamp: "2020-05-28T11:11:42" 
11:11:52.519 [error] LASPVIN Recevied delta took: 234 microseconds at TimeStamp: "2020-05-28T11:11:52" 
11:11:52.585 [error] LASPVIN Recevied delta took: 243 microseconds at TimeStamp: "2020-05-28T11:11:52" 
11:11:53.103 [error] LASPVIN Recevied delta took: 216 microseconds at TimeStamp: "2020-05-28T11:11:53" 
11:12:42.627 [error] LASPVIN Recevied delta took: 21849 microseconds at TimeStamp: "2020-05-28T11:12:42" 
11:12:52.532 [error] LASPVIN Recevied delta took: 222 microseconds at TimeStamp: "2020-05-28T11:12:52" 
11:12:52.597 [error] LASPVIN Recevied delta took: 243 microseconds at TimeStamp: "2020-05-28T11:12:52" 
11:12:53.118 [error] LASPVIN Recevied delta took: 235 microseconds at TimeStamp: "2020-05-28T11:12:53" 
11:12:59.043 [warning] global: 'a@10.0.0.250' failed to connect to 'c@11.0.0.252'
lasp_peer_service:stop().
11:13:09.457 [notice] "received stop request"
ok
(a@10.0.0.250)3> root@d1:/opt/lasp# 
root@d1:/opt/lasp# 
root@d1:/opt/lasp# exit
exit
root@csst-06:/home/ubuntu/laspdev/utility# exit
exit

Script done on 2020-05-28 11:13:13+0000
